lib-opt/GHCup/OptParse/Config.hs

```diff
@@ -96,8 +98,10 @@ configFooter = [s|Examples:
   ghcup config init
 
   # set <key> <value> configuration pair
-  ghcup config set <key> <value>|]
+  ghcup config set <key> <value>
 
+  # add a release channel
+  ghcup config add-release-channel vanilla|]
```

Comment:
Vanilla isn't a good example, because we usually want to set it instead of adding it.

So I think we want twe examples:

- adding prereleases
- setting vanilla (discrding everything else)

Reply:
fixed